src/bin/main.rs

```diff
@@ -18,7 +18,7 @@ fn main() {
 	let provider = DefaultAWSCredentialsProviderChain::new();
 	let region = Region::UsEast1;
 
-	let provider2 = ProfileCredentialsProvider::new();
+	let provider2 = ProfileCredentialsProvider::new().unwrap();
```

Comment:
I think we should use a matcher on the return type instead of unwrapping.  Show proper usage, etc...

For a different PR: change to DefaultAWSCredentialsProviderChain as well instead of forcing a profile provider.